js/views/standalone_registration_view.js

```diff
@@ -163,6 +164,18 @@
       await window.ConversationController.load();
       Whisper.RotateSignedPreKeyListener.stop(Whisper.events);
     },
+    async cancelSecondaryDevice() {
+      Whisper.events.off(
+        'secondaryDeviceRegistration',
+        this.onSecondaryDeviceRegistered
+      );
+      this.$('#register-secondary-device')
+        .removeAttr('disabled')
+        .text('Link');
```

Comment:
Not sure how consistent we are being with using the i18n stuff or if we are just resigning to doing a big sweep up in future when we want to do translations?

Reply:
You're right, I guess we'll have to do a big sweep...